txtdirect.go

```diff
@@ -337,7 +337,7 @@ func Redirect(w http.ResponseWriter, r *http.Request, c Config) error {
 		if err != nil {
 			return err
 		}
-		reverseProxy := proxy.NewSingleHostReverseProxy(u, "", proxyKeepalive, proxyTimeout)
+		reverseProxy := proxy.NewSingleHostReverseProxy(u, "", proxyKeepalive, proxyTimeout, fallbackDelay)
```

Comment:
This line returned a ``too many arguments`` error at first for me so I've removed the fallback argument to make it work.
It happened because my local caddy package was not updated, they have added it in mholt/caddy#2309